src/style.css

```diff
@@ -1,3 +1,95 @@
-body {
+* {
   margin: 0;
+  padding: 0;
 }
+
+:root {
+  --blue: #00ACDC;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-right: 50px;
+}
+
+header:hover {
+  border: 3px solid var(--blue);
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  width: 90px;
+  height: 60px;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.img {
+height: 40px;
+}
+
+.img:hover {
+  border: 3px solid var(--blue);
+  cursor: none;
+  cursor: default;
+
+}
+
+nav {
+  width: 641px;
+}
+
+.li {
+  text-decoration: none;
+
+}
+
+.li:hover {
+  border-bottom: 1px solid var(--blue);
+  cursor: default;
+}
+
+ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 60px;
+  align-items: center;
+}
```

Comment:
<img width="1135" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/959d5ba6-ac4b-4eaa-90bf-211f46164bbc">

Reply:
fix it through all code